Read-and-save-PowerPoint-presentation/Open-and-save-PowerPoint/Google-Drive/Save-PowerPoint-document/Save-PowerPoint-document/Program.cs

```diff
@@ -0,0 +1,109 @@
+﻿using Google.Apis.Auth.OAuth2;
+using Google.Apis.Drive.v3;
+using Google.Apis.Services;
+using Google.Apis.Util.Store;
+using File = Google.Apis.Drive.v3.Data.File;
+using Syncfusion.Presentation;
+
+namespace Save_PowerPoint_document
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            //Create a new instance of PowerPoint Presentation file
+            IPresentation pptxDocument = Presentation.Create();
+
+            //Add a new slide to file and apply background color
+            ISlide slide = pptxDocument.Slides.Add(SlideLayoutType.TitleOnly);
+
+            //Specify the fill type and fill color for the slide background 
+            slide.Background.Fill.FillType = FillType.Solid;
+            slide.Background.Fill.SolidFill.Color = ColorObject.FromArgb(232, 241, 229);
+
+            //Add title content to the slide by accessing the title placeholder of the TitleOnly layout-slide
+            IShape titleShape = slide.Shapes[0] as IShape;
+            titleShape.TextBody.AddParagraph("Company History").HorizontalAlignment = HorizontalAlignmentType.Center;
+
+            //Add description content to the slide by adding a new TextBox
+            IShape descriptionShape = slide.AddTextBox(53.22, 141.73, 874.19, 77.70);
+            descriptionShape.TextBody.Text = "IMN Solutions PVT LTD is the software company, established in 1987, by George Milton. The company has been listed as the trusted partner for many high-profile organizations since 1988 and got awards for quality products from reputed organizations.";
+
+            //Add bullet points to the slide
+            IShape bulletPointsShape = slide.AddTextBox(53.22, 270, 437.90, 116.32);
+
+            //Add a paragraph for a bullet point
+            IParagraph firstPara = bulletPointsShape.TextBody.AddParagraph("The company acquired the MCY corporation for 20 billion dollars and became the top revenue maker for the year 2015.");
+
+            //Format how the bullets should be displayed
+            firstPara.ListFormat.Type = ListType.Bulleted;
+            firstPara.LeftIndent = 35;
+            firstPara.FirstLineIndent = -35;
+
+            //Add another paragraph for the next bullet point
+            IParagraph secondPara = bulletPointsShape.TextBody.AddParagraph("The company is participating in top open source projects in automation industry.");
+
+            //Format how the bullets should be displayed
+            secondPara.ListFormat.Type = ListType.Bulleted;
+            secondPara.LeftIndent = 35;
+            secondPara.FirstLineIndent = -35;
+
+            //Gets a picture as stream
+            FileStream pictureStream = new FileStream(Path.GetFullPath("../../../Output/Image.jpg"), FileMode.Open);
+
```

Comment:
Please changes image file path properly. 
Path.GetFullPath("../../../Data/Image.jpg")

Reply:
Modified the path properly.